src/class-methods/class-private-method-whitespace-usage-syntax-error.case

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2018 Bloomberg LP. All rights reserved.
```

Comment:
@leobalter I potentially need to add more cases like this one which handle the usage of private fields/methods/accessors/async methods/generators etc with the whitespace syntax error. it'll potentially lead to a lot of .case files. It might be a lot but also wondering if there is a better way.

Reply:
also, would recommend putting some of this in function-forms folder?

Reply:
the class-fields should hold all the cases for checking invalid whitespace between the sigil and identifier name. I'm not sure how this would be covered outside.